DataAccess/Public/Include/DataAccess/IServerRepositories.hpp

```diff
@@ -53,22 +94,45 @@ struct IRepliesRepository : IMasterRepository
  */
 struct IChannelsRepository : IMasterRepository
 {
-    /// Virtual method to get channel List
+    /**
+    * @brief  method for getting all channel list from the server.
+    * @return channels list as vector of ChannelInfo.
+    */
     virtual std::vector<Models::ChannelInfo> getAllChannelsList() = 0;
-    /// Virtual method to delete channel
+
+    /**
+    * @brief  method for deleting channel.
+    * @return special code for success or failed deleting;
+    */
     virtual Utility::ChannelDeleteCode deleteChannel(const Models::ChannelDeleteInfo& channel) = 0;
 
-    /// Virtual method to create channel
+    /**
+    * @brief  method for creating channel.
+    * @return special code for success or failed creating;
```

Comment:
That's a code smell which means that using at the end of comment line ';' is a bad idea because it's not that doxygen can detect as documentable symbol